pkg/ccn-lite/ccn-lite-riot.h

```diff
@@ -12,6 +12,7 @@
 /**
  * @defgroup    pkg_ccnlite CCN-Lite stack
  * @ingroup     pkg
+ * @ingroup     net
```

Comment:
IMHO adding a second group here is not a good idea, these are third party, external packages, adding additional grouping like `net`, `sys`, `sys_fs` or any other RIOT core grouping will mix things up in the documentation.

I know I introduces this _features_ of multiple groups for `cpu/periph/` drivers, but these are RIOT internal, hence that fits.